src/install/python/python-agent-non-web-apps.mdx

```diff
@@ -146,7 +146,7 @@ Due to the limitation on what Django management commands can be monitored, you n
 instrumentation.scripts.django_admin = syncdb sqlflush
 ```
 
-By default, the startup timeout is automatically specified to be 10.0 seconds when monitoring the Django management commands. If you need to override the startup timeout, you can set the `instrumentation.background_task.startup_timeout` setting within the same `import-hook:django` configuration section.
+By default, the startup timeout is automatically specified to be 10.0 seconds when monitoring the Django management commands. This means New Relic will block the command for up to 10s giving New Relic a chance to start up so it can monitor the command, otherwise it would not be able to start in time and would not be able to instrument the command. If you need to override the startup timeout, you can set the `instrumentation.background_task.startup_timeout` setting within the same `import-hook:django` configuration section.
```

Comment:
Replace the comma after "command" with a semicolon or just make it its own sentence